docker-compose.yml

```diff
@@ -451,6 +451,14 @@ services:
       AWS_SECRET_KEY: 'qwerty'
       AWS_SECRET_ACCESS_KEY: 'qwerty'
 
+  save-and-return-simulator:
+    container_name: save-and-return-simulator
```

Comment:
Another container! 🤣

Reply:
Yeah, we've been talking about how to reduce the number ... :)

Reply:
Oh, sweet irony! :D